src/css/components/_live.scss

```diff
@@ -3,12 +3,27 @@
 .video-js .vjs-live-control {
   @include display-flex(flex-start);
   @include flex(auto);
-  font-size: 1em;
-  line-height: 3em;
+  font-size: 1.5em;
```

Comment:
The text was very small in relation to the icons on the control bar, I bumped it up a tiny bit.

Reply:
Unfortunately, this means that the LIVE display is now inconsistent with the size of other text in the control bar like the various time displays. I would leave this as is and we should open an issue to look into increase the text size.

Reply:
Created one from the comment #5500

Reply:
ok